terralab/terralab

```diff
@@ -1,6 +0,0 @@
-#!/usr/bin/env poetry run python
```

Comment:
so when someone is developing, how do they make sure they can test on the latest code changes.  do they have to keep running `poetry install` so they can use `terralab` with the latest code or once they run `poetry install` once, `terralab` will always point to their latest code?

Reply:
nope, it's pretty magical, you can run `poetry install` the one time and then keep developing and when you run `terralab whatever` it uses the updated code. (i tested this when doing the updates to the version stuff that's also in this pr)

Reply:
thats super awesome